tests/test_dependencies.py

```diff
@@ -11,8 +11,12 @@
                                  is_local_source)
 import sacred.optional as opt
 
-EXAMPLE_SOURCE = 'tests/__init__.py'
-EXAMPLE_DIGEST = '9e428c0aa58b75ff150c4f625e32af68'
+TEST_DIRECTORY = os.path.dirname(__file__)
+EXAMPLE_SOURCE = os.path.join(TEST_DIRECTORY, '__init__.py')
+
+# The digest below is calculated from the test/__init__.py with only Python shebang and coding-information.
+# This type of hard-coding is most probably a quite bad idea.
```

Comment:
Passive agressive bugfixing. I like it :wink:

Reply:
But I'm afraid you miscalculated the hash. Could you revert it to `9e428c0aa58b75ff150c4f625e32af68` ?

Reply:
Exactly what I was I afraid of, that's the hash I got on my Windows work computer.

Reply:
Interesting, I wonder how this could ever pass on AppVoyer ever :D